docs/language/builtins.rst

```diff
@@ -237,15 +237,15 @@ ABI encodes the arguments to bytes. Any number of arguments can be provided.
 
 On Substrate, foo will be ``hex"f100"``. On Ethereum this will be ``hex"00000000000000000000000000000000000000000000000000000000000000f1"``.
 
-abi.encodeWithSelector(bytes4 selector, ...)
+abi.encodeWithSelector(selector, ...)
```

Comment:
This should have a note saying that the selector is know as a discriminator on Solana.